### PR TITLE
Regenerate cert with right id-DMTF-hardware-identity format

### PR DESCRIPTION
**Pull request by Wenxing-hou**
Fix: #2325

Regenerate all related cert with right id-DMTF-hardware-identity format.

### CODE REVIEW

File: unit_test/sample_key/openssl.cnf [hunk 1]
```diff
@@ -12,7 +12,11 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 subjectKeyIdentifier = hash
 subjectAltName = otherName:1.3.6.1.4.1.412.274.1;UTF8:ACME:WIDGET:1234567890
 extendedKeyUsage = critical, serverAuth, clientAuth, OCSPSigning
-1.3.6.1.4.1.412.274.6 = ASN1:OID:1.3.6.1.4.1.412.274.2
+1.3.6.1.4.1.412.274.6 = ASN1:SEQUENCE:id_spdm_cert_oids
+[id_spdm_cert_oids]
+field1 = SEQUENCE:id_spdm_cert_oid
+[id_spdm_cert_oid]
+field1 = OID:1.3.6.1.4.1.412.274.2
```
Comment (jyao1) [inline unit_test/sample_key/openssl.cnf:19]: This seems good to me.

Comment (rioone2023) [reply unit_test/sample_key/openssl.cnf:19]: I checked "openssl.cnf" and some of the certificates generated using it. These are the expected modifications and no problem.

Comment (jyao1) [reply unit_test/sample_key/openssl.cnf:19]: Thanks! Approve it.

**Review by jyao1 — APPROVED** [review]

**Comment by steven-bellock** [comment]
@jyao1 feel free to merge as you see fit.

**Pull request merged**